events-api/src/modules/tickets/dto/payment-ticket.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IsNotEmpty, IsNumber, IsString, Length, Matches, Validate } from "class-validator";
+import { CardExpiryConstraint } from '../../../shared/validators/cardValidators/card-expiry-date.validator';
+import { CardHolderNameConstraint } from '../../../shared/validators/cardValidators/card-holder-name.validator';
```

Comment:
This decorator part of the ticket module. It is not shared among other modules, descope it to the tickets module